apps/api/src/routes/agents.routes.ts

```diff
@@ -82,35 +82,7 @@ export function configureAgentsRoutes(
    *                 agents:
    *                   type: array
    *                   items:
-   *                     type: object
-   *                     properties:
-   *                       id:
-   *                         type: string
-   *                         format: uuid
-   *                       ownerId:
-   *                         type: string
-   *                         format: uuid
-   *                       walletAddress:
-   *                         type: string
-   *                         example: "0x1234567890abcdef1234567890abcdef12345678"
-   *                       name:
-   *                         type: string
-   *                         example: "Trading Bot Alpha"
-   *                       description:
-   *                         type: string
-   *                         example: "AI agent focusing on DeFi yield farming"
-   *                       imageUrl:
-   *                         type: string
-   *                         example: "https://example.com/bot-avatar.jpg"
-   *                       status:
-   *                         type: string
-   *                         enum: [active, suspended, deleted]
-   *                       createdAt:
-   *                         type: string
-   *                         format: date-time
-   *                       updatedAt:
-   *                         type: string
-   *                         format: date-time
+   *                     $ref: '#/components/schemas/Agent'
```

Comment:
You'll see all the different places an agent was spec'ed out now have a `$ref` to the one spec.